todo/src/stores/store.js

```diff
@@ -0,0 +1,5 @@
+import { createStore } from 'redux';
+import { todosReducer as todos } from '../reducers/reducer';
```

Comment:
I understand what you are trying to do here but readability should trump shortness. Keep todoReducer*

* check previous comment

Reply:
Okay